nucliadb/tests/search/integration/api/v1/test_search.py

```diff
@@ -40,6 +40,7 @@
 RUNNING_IN_GH_ACTIONS = os.environ.get("CI", "").lower() == "true"
 
 
+@pytest.mark.flaky(reruns=5)
```

Comment:
Restored, I accidentally removed them in the previous PR.

Reply:
I thought it was on purpose :)